### PR TITLE
WIP: 0.26 Changelog

### PR DESCRIPTION
**Pull request by calebhailey**
Fixes/implements #414

### CODE REVIEW

File: docs/0.26/overview/changelog.md [hunk 1]
```diff
@@ -10,209 +10,197 @@ weight: 5
...
-- The legacy/deprecated Sensu API singular resources (e.g. `/check/:check_name`
-  instead of `/checks/:check_name`), have been removed. Singular resources were
-  never documented and have not been used by most community tooling (e.g.
-  Uchiwa) since the very early Sensu releases (circa 2011-2012).
-
-### CHANGES {#core-v0-25-0-changes}
+- Event silencing is now built-in to Sensu Core, and a new `/silenced` API is
+  now available, which may conflict with the legacy "silence stash" filtering
+  provided by Sensu Plugin handlers that are based on the `sensu-plugin` gem. A
+  new `"handle_silenced": true` attribute/override is available to disable this
```
Comment (calebhailey) [inline docs/0.26/overview/changelog.md]: @cwjohnston should we also refer to the `"use_deprecated_filtering": false` check definition attribute here?

File: docs/0.26/overview/changelog.md [hunk 2]
```diff
@@ -10,209 +10,212 @@ weight: 5
...
+  definitions for handlers that rely on "silence stashes".
+
+  _NOTE: Event filtering is moving out of the Sensu Plugins project and into
+  Sensu Core. Though "silence stashes" are still supported in the `sensu-plugin`
+  gem version 1.4, this functionality will be deprecated in an upcoming release
+  (along with all built-in filters in `sensu-plugin`). Please refer to the
+  [Deprecating Event Filtering in sensu-plugin][?] blog post for more
+  information._  
+
+- Handler `subdue` definitions are no longer supported. Time-based filtering is
```
Comment (portertech) [inline docs/0.26/overview/changelog.md]: The handler definition attribute `subdue` is no longer supported.

File: docs/0.26/overview/changelog.md [hunk 3]
```diff
@@ -10,209 +10,212 @@ weight: 5
...
+  `subdue` definitions are now supported by standalone checks.
+
+  By way of comparison, the legacy `subdue` definition specification was
+  limited to a single time window rule, with an array of exceptions. This was
+  not only confusing, it made it very difficult to apply a simple "don't execute
+  this check outside of 9-5, M-F" rule.
 
-### CHANGES {#core-v0-25-1-changes}
+  ~~~ json
+  {
```
Comment (portertech) [inline docs/0.26/overview/changelog.md]: Perhaps we should provide the rest of the configuration scope, e.g. check definition subdue, for more context?

Comment (calebhailey) [reply docs/0.26/overview/changelog.md]: Yup.

**Comment by calebhailey** [comment]
We can't finish this (adding links to reference docs) until #416, #417, #418, #419, #420, and #421 are completed.

File: docs/0.26/overview/changelog.md (hunk 1 above)
Comment (cwjohnston) [reply docs/0.26/overview/changelog.md]: @calebhailey good idea. I am thinking something like:

```
The Sensu embedded Ruby environment now includes sensu-plugin 1.4.2.
As described [on the Sensu blog](http://bit.ly/sensu-plugin) this version
of the library will print deprecation warnings by default. Set the check
attribute `enable_deprecated_filtering: false` to disable the
deprecated filtering behavior.
```

**Comment by cwjohnston** [comment]
Updated with links to reference docs and github issues + PRs

**Comment by cwjohnston** [comment]
I think this changelog is ready to merge.

**Comment by portertech** [comment]
efd2987e-3867-11e4-9b56-e9145249548a

**Pull request merged**